docker-compose.apps.yml

```
@@ -204,4 +204,5 @@ services:
     ports:
       - "8000:8000"
 volumes:
-  kafka-data:
\ No newline at end of file
+  user-dev-db-data:
+    driver: local
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)